packages/slate/src/changes/at-current-range.js

```diff
@@ -192,6 +192,10 @@ Changes.insertText = (change, text, marks) => {
   marks = marks || selection.marks || document.getInsertMarksAtRange(selection)
   change.insertTextAtRange(selection, text, marks)
 
+  if (selection.isExpanded) {
+    change.select(change.value.selection.collapseToStart(), { snapshot: false })
+  }
+
```

Comment:
Can you add a test that verifies this logic is working?

Reply:
The reason is that, in `insertTextAtRange` or in `deleteAtRange`, the result selection can be 
```
{
anchorKey: anchorText,
anchorOffset: anchorText.text.length,
focusKey: getNextText(anchorText),
focusOffset: 0
}
```
, which is a `range.isExpanded === true`

I have tested it in `:8080/dev.html`.
Shall I add two tests both in `insertTextAtRange` and `insertText` with {normalize: false} to ensure that
1. `insertTextAtRange` renders isExpanded:true
2. `insertText` renders isExpanded:false

Reply:
Test added~